odd_collector/adapters/sqlite/adapter.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from funcy import concat, lpluck_attr
+from odd_collector_sdk.domain.adapter import AbstractAdapter
+from odd_models.models import DataEntityList
+from oddrn_generator import SQLiteGenerator
+
+from odd_collector.domain.plugin import SQLitePlugin
+from .mappers.database import map_database
+from .mappers.table import map_table
+from .mappers.view import map_view
+from .repository.base_repository import Repository
+from .repository.sqlalchemy_repository import SqlAlchemyRepository
+
+
+class Adapter(AbstractAdapter):
```

Comment:
Use BaseAdapter instead for derivation. Take a look at postgres adapter

Reply:
Fixed